server/resp/src/main/java/org/infinispan/server/resp/meta/ClientMetadata.java

```diff
@@ -0,0 +1,109 @@
+package org.infinispan.server.resp.meta;
+
+import java.util.concurrent.atomic.LongAdder;
+
+/**
+ * Metadata for `<code>INFO</code>` command.
+ * <p>
+ * Holds metadata about client connections. All the values are specific to a single node.
+ * </p>
+ *
+ * @since 15.1
+ * @see org.infinispan.server.resp.commands.INFO
+ */
+public final class ClientMetadata {
+
+   /**
+    * Number of client connections
+    */
+   private final LongAdder connectedClients = new LongAdder();
```

Comment:
I would highly recommend using `transient long` fields and an `AtomicLongFieldUpdater` to update those values. It will cut down the size of this class by over half.

These fields should not be updated that frequently, so I don't think it is required for these to be LongAdders.

Reply:
Updated.